### DISCUSSION TITLE
can we pass on the split in the prompt

### DISCUSSION THREAD

**Discussion by parikannappan** [sinaptik-ai/pandas-ai#398] [General]
Can we do column split like we do in pandas using pandas using prompt  , example "str.split('To', expand=True)" .
if yes where do we set the new column name?

**Comment by gventuri** [comment]
@parikannappan sure thing. I would ask it like this:
`split the column "your_column" in 2 columns using "To" as a separator and name the new column "your_new_column"`

Please try it out and let me know :)
